### DISCUSSION TITLE
Mirth 4.2 consuming all CPU

### DISCUSSION THREAD

**Discussion by jacsonsoares** [nextgenhealthcare/connect#5643] [Q&A]
Hi all, 

I'm using Mirth 4.2 on Windows 11 and it's consuming all CPU.
This did not happen in previous versions.
I'm using the same java:
openjdk version "11.0.12" 2021-07-20
OpenJDK Runtime Environment Microsoft-25199 (build 11.0.12+7)
OpenJDK 64-Bit Server VM Microsoft-25199 (build 11.0.12+7, mixed mode)

I have two co-workers who have the same problem.
Does anyone have this problem or know the solution?

Thanks for helping me

![MirthUsageCPU](https://user-images.githubusercontent.com/11412979/215494527-ca617443-0de9-4144-b0ed-920a6225cd3e.png)

**Comment by jonbartels** [comment]
Take a thread dump - https://stackoverflow.com/questions/2124672/java-stack-dump-on-windows

Once you have that you can analyse it and see what threads are running or if there are deadlocked threads.

The thread dump will not contain PII/PHI but it will contain channel names. It should be safe enough to post online but you need to exercise good judgement.
